### PR TITLE
Switch click packaging to CMake

### PR DESCRIPTION
**Pull request by jonnius**

### CODE REVIEW

File: src/CMakeLists.txt [hunk 1]
```diff
@@ -30,6 +30,14 @@ if(MAPMATCHING_AVAILABLE)
...
 endif()
 
+if(S2INCLUDES)
+    include_directories(${S2INCLUDES})
+endif()
+
+if(S2LIBS)
+    link_directories(${S2LIBS})
+endif()
+
```
Comment (rinigus) [inline src/CMakeLists.txt:40]: Do we need to move S2 section here? I am mainly wondering and trying to understand the reason behind it.

**Review by rinigus — COMMENTED** [review]
Just a comment regarding move of one section in src/CMakeLists.txt

File: src/CMakeLists.txt (hunk 1 above)
Comment (jonnius) [reply src/CMakeLists.txt:40]: Without that move, the link directory is not passed to the compiler resulting in the linker not finding the lib. It seems CMake only considers link directories defined before creating the executable.

Comment (rinigus) [reply src/CMakeLists.txt:40]: Thanks! Resolving and merging then

**Pull request merged**